doc/articles/faq.md

```diff
@@ -33,7 +33,9 @@ You can develop Uno Platform applications on Windows, macOS, or Linux. Supported
 
 Much like the new UI technologies from Microsoft, Uno Platform doesn’t support creation of new applications using VB.NET.
 
-If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic, packaged as NET standard libraries into a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#. Additionally, If you’d like to move any of your VB.NET code to C# you may be able to use automated tools such as https://converter.telerik.com
+If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic, packaged as NET standard libraries into a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#.
```

Comment:
```suggestion
If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic. I needs to be built as .NET Standard libraries, then used in a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#.
```

Reply:
Ok, although this comment is mis-addressed :)  Your remark is, in fact, to duke7553, as this sentence is from his commit dated Jan 12.

Reply:
No worries. Let's apply it anyways :)